analyses/malawi/dryspells_trigger/01_chirps_dry_spell_detection.R

```diff
@@ -356,7 +356,7 @@ dry_spells_during_rainy_season_list <- dry_spells_during_rainy_season_list %>% m
 # save full list of dry spells
 full_list_dry_spells <- dry_spells_details %>%
                           left_join(rainy_seasons[, c('pcode', 'season_approx', 'onset_date', 'cessation_date')], by = c('pcode', 'season_approx'), all.x = T, all.y = T) %>% # add rainy onset and cessation dates
-                          mutate(during_rainy_season = ifelse(dry_spell_confirmation >= onset_date & dry_spell_confirmation <= cessation_date, 1, 0)) %>% # identifies dry spells that reached 14-d rolling sum (confirmation_date) during rainy season
+                          mutate(during_rainy_season = ifelse(dry_spell_confirmation >= onset_date & dry_spell_confirmation <= cessation_date, 1, 0)) %>% # identifies dry spells that reached 14-d rolling sum (confirmation_date) during rainy season. NA = start/end of rainy season unknown
```

Comment:
NA also occurs for values with `season_approx` being `outside_rainy_season`

Reply:
My solution was to assign a season_name (reflecting season_approx) only for records for which during_rainy_season == 1 (based on onset/cessation dates). It should keep the NAs under `during_rainy_season`, as discussed. Let me know if you think that doesn't work.